doc/source/ecosystem.rst

```diff
@@ -130,6 +130,13 @@ API
 Quandl API for Python wraps the Quandl REST API to return
 Pandas DataFrames with timeseries indexes.
 
+`pydatastream <https://github.com/vfilimonov/pydatastream>`_
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Can you make this line at least as long as the title above it? (otherwise sphinx raises warnings ..)